src/Button/makeStories.js

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment } from 'react';
+import { action } from '@storybook/addon-actions';
+import { boolean, text, select } from '@storybook/addon-knobs';
+import theme from '../theme';
+
+export default (storiesOf, { Button }) =>
+  storiesOf('Button', module)
+    .add('playground', () => (
+      <Button
+        color={select('color', Object.keys(theme.color), 'primary')}
+        kind={select('kind', ['filled', 'outlined'], 'filled')}
+        shape={select('shape', ['bluntEdged', 'sharpEdged', 'capsular', 'circular'], 'bluntEdged')}
+        size={select('size', ['small', 'medium', 'large'], 'medium')}
+        block={boolean('block', false)}
+        disabled={boolean('disabled', false)}
+        onClick={action('button clicked')}
+      >
+        {text('children', 'Button')}
+      </Button>
+    ))
+    .add('kinds', () => (
+      <Fragment>
+        <Button kind="filled">
+        Filled Button
+        </Button>
+        <br /><br />
```

Comment:
Sadly the `br` tag won't work in native, replacing this with `Spacer` should work across :)

Reply:
Gotcha 🙈